plugin/api.thrift

```diff
@@ -130,6 +130,28 @@ struct Argument {
      * Argument type.
      */
     2: required Type type
+    /**
+     * Annotations defined on this type.
+     *
+     * Note that these are the Thrift annotations listed after the type
+     * declaration in the Thrift file.
```

Comment:
Annotations defined on this argument.

    Note that these are listed after the parameter or exception name.